src/recorder/stream_recorder/stream_recorder_options.h

```diff
@@ -64,7 +64,7 @@ struct StreamRecorderOptions {
   // after a failure.
   std::chrono::microseconds satellite_failure_retry_period =
       std::chrono::duration_cast<std::chrono::microseconds>(
-          std::chrono::seconds{5});
+          std::chrono::seconds{1});
```

Comment:
Please make this a constant somewhere.

Reply:
This file has all the internal stream recorder options inline like that https://github.com/lightstep/lightstep-tracer-cpp/blob/http-streaming/src/recorder/stream_recorder/stream_recorder_options.h

Is there really much to gain by moving them to an extra variable constant?